packages/core-ethereum/src/chainInteractions.ts

```diff
@@ -0,0 +1,22 @@
+import BN from 'bn.js'
```

Comment:
we are doing something similar in `src/ethereum.ts`

Reply:
ah ok, we can merge

Reply:
feel free to kill it, this is more useful